Automattic/wp-calypso
# List Views: Correct Copy Link styling.

This PR makes the styling for the Copy Post menu item a little more resilient to `.button` changes elsewhere.

**Master**
<img width="1010" alt="Screen Shot 2019-10-31 at 1 56 10 PM" src="https://user-images.githubusercontent.com/349751/67985614-74ac2780-fbe6-11e9-81f3-bcfadf89410c.png">

**This branch**
<img width="1036" alt="Screen Shot 2019-10-31 at 1 57 24 PM" src="https://user-images.githubusercontent.com/349751/67985608-6eb64680-fbe6-11e9-9b43-3d38bf41d097.png">


#### Testing instructions

1. Load this branch locally or with `calypso.live`.
2. Go to a site's posts or pages list view (eg. `/posts/my`).
3. Open an ellipsis menu, and verify the Copy Link item is styles correctly (on hover too).
